browser-extension/src/ui/styles.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Theme, createStyles } from '@material-ui/core';
+
+export const styles = (theme: Theme) =>
+  createStyles({
+    appBar: {
+      position: 'relative',
+    },
+    layout: {
+      width: 'auto',
+      marginLeft: theme.spacing(2),
+      marginRight: theme.spacing(2),
+      [theme.breakpoints.up(400 + theme.spacing(2) * 2)]: {
+        width: 800,
```

Comment:
what if user browser width is 700 ?, should this depend on window width too ?

Reply:
I copied&pasted that to get started with styling from material-ui (since I am not really good at those things...). Having this makes no sense, I removed it.